pkg/registry/servicecatalog/binding/storage.go

```diff
@@ -135,6 +137,38 @@ func NewStorage(opts server.Options) (rest.Storage, rest.Storage, error) {
 		DeleteStrategy:          bindingRESTStrategies,
 		EnableGarbageCollection: true,
 
+		TableConvertor: tableconvertor.NewTableConvertor(
+			[]metav1beta1.TableColumnDefinition{
+				{Name: "Name", Type: "string", Format: "name"},
+				{Name: "Service-Instance", Type: "string"},
+				{Name: "Secret-Name", Type: "string"},
+				{Name: "Status", Type: "string"},
+				{Name: "Age", Type: "string"},
+			},
+			func(obj runtime.Object, m metav1.Object, name, age string) ([]interface{}, error) {
```

Comment:
I've got a bunch of logic just like this in the svcat cli too. Maybe we can consolidate in a follow-on issue?

https://github.com/kubernetes-incubator/service-catalog/blob/master/pkg/svcat/service-catalog/binding.go#L198

https://github.com/kubernetes-incubator/service-catalog/blob/master/cmd/svcat/output/binding.go#L29